Validations.js

```diff
@@ -0,0 +1,35 @@
+function nameRequired(options) {
+  return _.isString(options.name);
+}
+
+function instrumentRequired(options) {
+  return _.isString(options.guitar) || _.isString(options.drums);
```

Comment:
this seems to be the same code as in nameRequired except with a different property name. If you were to rename nameRequired to stringRequired and it would receive the attribute as a parameter you would have no more duplication.